### PR TITLE
COMPONENT: Input Field

### PR DESCRIPTION
**Pull request by j-eick**
Simple Input Field for prompts. 

Simple animation that makes conditional Send-Button (dis-)appear. Will change it in next iteration.

### CODE REVIEW

**Review by Elohim2598 — APPROVED** [review]
LGTM

**Review by adan-ayaz-stan — COMMENTED** [review]
PR has changes from the `landingpage` branch. As such, it will be merged after `landingpage` branch is merged so differences are more evident and conflicts can be avoided.

PR on hold.

File: frontend/package-lock.json
Comment (adan-ayaz-stan) [inline frontend/package-lock.json:1]: File needs to be deleted.

File: frontend/src/components/ui/heading/Heading.svelte
Comment (adan-ayaz-stan) [inline frontend/src/components/ui/heading/Heading.svelte:1]: `border-b` instead of `border-b-[1px]`

File: frontend/src/components/ui/input/Input.svelte
Comment (adan-ayaz-stan) [inline frontend/src/components/ui/input/Input.svelte:29]: remove empty function

File: frontend/src/components/ui/input/Input.svelte
Comment (adan-ayaz-stan) [inline frontend/src/components/ui/input/Input.svelte:30]: arbitary values are bad practice - find tailwind equivalents

File: frontend/src/components/ui/input/Input.svelte
Comment (adan-ayaz-stan) [inline frontend/src/components/ui/input/Input.svelte:23]: `border-y-2` instead of `border-b-2 border-t-2`

File: frontend/src/components/ui/input/Input.svelte
Comment (adan-ayaz-stan) [inline frontend/src/components/ui/input/Input.svelte:17]: same comment as below

File: frontend/src/routes/+page.svelte
Comment (adan-ayaz-stan) [inline frontend/src/routes/+page.svelte:1]: Import the app.css in the root layout. There's a (landing) routes folder with it's own layout that contains the header and footer.

File: frontend/src/routes/team/+page.svelte
Comment (adan-ayaz-stan) [inline frontend/src/routes/team/+page.svelte:27]: Bad practice. Upload the image to an image upload service like Cloudinary.

**Review by adan-ayaz-stan — CHANGES_REQUESTED** [review]

File: frontend/src/routes/team/+page.svelte
Comment (Elohim2598) [reply frontend/src/routes/team/+page.svelte:27]: This was already addressed and PR was already merged to main, so might not be needed

File: frontend/src/routes/+page.svelte
Comment (Elohim2598) [reply frontend/src/routes/+page.svelte:1]: Already done in the merged PR, so might not be necessary here

**Pull request merged**
